src/gui/sdlmain.cpp

```diff
@@ -251,6 +251,55 @@ enum PRIORITY_LEVELS {
 	PRIORITY_LEVEL_HIGHEST
 };
 
+class pp_scale_t {
+public:
+	pp_scale_t(const int source_w,
+	           const int source_h,
+	           const double aspect_ratio,
+	           const bool source_is_doubled,
```

Comment:
Is the `is` necessary? Perhaps just `source_doubled`?

Reply:
Bool is the one unique type that holds metadata about something instead of the actual thing. One of the styles we've tried to add is always including a boolean-ess adjective to functions and variables that are or return book (instead of the actual thing). 

Take these:

- aspect_ratio
- doubled_source

You can't tell if those hold the actual aspect ratio and actual 'source' or if they're bools. But then.. 

- have_aspect_ratio
- source_is_doubled

Immediately disambiguates them.  Just a few extra characters might a big difference.

Reply:
Lots of words can be used to add this booleaness and some even help add intent as well: have, should, wants, is, prefers,  ... (Surely lots more)

Reply:
Fair enough. Carry on.